app/components-react/root/LiveDock.tsx

```diff
@@ -284,16 +273,21 @@ function LiveDock(p: { onLeft: boolean }) {
       'chatTabs',
       'applicationLoading',
       'hideStyleBlockers',
-      'hideViewerCount',
-      'viewerCount',
       'pageSlot',
       'canAnimate',
+      'currentViewers',
       'liveText',
       'isPopOutAllowed',
       'streamingStatus',
     ]),
   );
 
+  const liveDockSize = useRealmObject(Services.CustomizationService.state).livedockSize;
+  const collapsed = useRealmObject(Services.CustomizationService.state).livedockCollapsed;
+  const hideViewerCount = useRealmObject(Services.CustomizationService.state).hideViewerCount;
```

Comment:
I think it's pretty clear that I should add some sort of selector API like we had for `useVuex` or similar to simplify this a bit.